jck/jck.mk

```diff
@@ -62,7 +62,7 @@ ifndef JCK_ROOT
 endif
 
 ifndef CONFIG_ALT_PATH
-  export CONFIG_ALT_PATH:=$(TEST_ROOT)$(D)jck$(D)/jtrunner/config/default
+  export CONFIG_ALT_PATH:=$(TEST_ROOT)$(D)jck$(D)jtrunner/config/default
```

Comment:
Thanks for this update. Since it's being fixed here, could we get all the `/` replaced with `$(D)` in `CONFIG_ALT_PATH` ?